content/blogs/2023-06-21-locxia-kestra.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Locxia: Processing Satellite Imagery with Kestra"
+description: How Ntico Leverage Kestra Orchestrator for their data platform Locxia.
+date: 2023-06-22T18:00:00
+category: Solutions
+author:
+  name: Benoit Pimpaud
+  image: "bpimpaud"
+image: /blogs/2023-06-22-locxia-kestra.jpg
+---
+
+Ntico has designed [Locxia](https://www.locxia.com/), a powerful platform that gives businesses and local authorities a deeper understanding of their territories and residents thanks to satellite imagery. At the core of their solution, they leveraged Kestra for its features and efficient management of large-scale data, including ingesting, processing, and visualizing vast datasets. 
```

Comment:
```suggestion
[Locxia](https://www.locxia.com/), developed by Ntico, is a robust platform that gives businesses and local authorities a deeper understanding of their territories and residents thanks to satellite imagery. At the core of their solution lies Kestra, used for ingesting, processing, and visualizing vast datasets. 
```

Reply:
I keep the active tense for the first sentence; but applied the rest ;)